workspaces/ui-v2/src/entry-points/cloud/TopLevelRoutes.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import CloudViewer from '../../spectacle-implementations/cloud-viewer';
+import WelcomePage from '../../optic-components/onboarding/WelcomePage';
+
+export default function TopLevelRoutes() {
+  return (
+    <Switch>
+
+      <Route
+        strict
+        //@TODO: centralize this path pattern
+        path="/public_specs/:specId"
```

Comment:
I believe in the past we've used `-` as a delimiter in urls as opposed to `_`. I believe that is more of an SEO thing but also consistency